frontend/src/components/forms/CreateLog.tsx

```diff
@@ -161,9 +241,9 @@ const CreateLog = () => {
       }),
```

Comment:
Also a bunch of defaulted -1's here, if there's a reason for this than you can keep it but otherwise I would make them empty strings